Src/CSharpier.Cli.Tests/CliTests.cs

```diff
@@ -38,7 +38,7 @@ public async Task Should_Format_Basic_File(string lineEnding)
         var formattedContent = "public class ClassName { }" + lineEnding;
         var unformattedContent = $"public class ClassName {{{lineEnding}{lineEnding}}}";
 
-        await WriteFileAsync("BasicFile.cs", unformattedContent);
+        await this.WriteFileAsync("BasicFile.cs", unformattedContent);
```

Comment:
If you want to always prepend `this.`, that's fine, but could you change all of them in a single PR?
Littering the changes all over the place makes it hard to review.

Reply:
Yeah sorry, I went ahead and merged it - https://github.com/belav/csharpier/pull/562